incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/openapi/http/proxy/OpenApiHttpProxyConfigGenerator.java

```diff
@@ -385,6 +388,117 @@ private <C> HttpRequestConfigBuilder<C> injectParams(
         return request;
     }
 
+    private <C> BindingConfigBuilder<C> injectHttpClientOptions(
+        BindingConfigBuilder<C> binding)
+    {
+        if (hasResponses())
+        {
+            binding.
+                options(HttpOptionsConfig::builder)
+                    .inject(this::injectHttpClientRequests)
+                    .build();
+        }
+        return binding;
+    }
+
+    private boolean hasResponses()
+    {
+        boolean result = false;
+        for (String pathName : openApi.paths.keySet())
+        {
+            PathView path = PathView.of(openApi.paths.get(pathName));
+            for (String methodName : path.methods().keySet())
+            {
+                Operation operation = path.methods().get(methodName);
+                if (hasResponses(operation))
+                {
+                    result = true;
+                    break;
+                }
+            }
+        }
+        return result;
+    }
```

Comment:
This method feels like it should be on `OperationView` instead of here.

Reply:
I managed to move both `hasRepsonses` methods out from here. It was necessary to create an `OperationView` and an `OperationsView` class (singular and plural). I think it makes sense, pls chk.